### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by isasmendiagus**

### CODE REVIEW

File: build/afterpack.js [hunk 1]
```diff
@@ -15,19 +15,19 @@ module.exports = async function (context) {
...
-  const launcherScript = path.join(
-    context.appOutDir,
-    'resources',
-    'launcher-script.sh'
-  );
-  // rename beekeeper-studio to beekeeper-studio-bin
-  await renameAsync(sourceExecutable, targetExecutable);
-  // rename launcher script to beekeeper-studio
-  await renameAsync(launcherScript, sourceExecutable);
+  // const executableName = context.packager.executableName;
```
Comment (mscasso-scanoss) [inline build/afterpack.js:18]: Why is this commented?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
-  "name": "scanoss-quickscan",
-  "version": "1.2.1",
+  "name": "quickscan-lite",
+  "version": "1.2.2",
+  "versiondate": "05.12.21",
```
Comment (mscasso-scanoss) [inline package.json:4]: This should be called buildDate, and It have to be updated automatically when the package is generated. Think about that for the next version.

File: package.json [hunk 3]
```diff
@@ -25,13 +26,14 @@
...
     "electron-about-window": "^1.13.4",
+    "isbinaryfile": "^4.0.8",
     "jquery": "^3.5.1",
     "popper.js": "^1.16.1",
     "sqlite3": "^5.0.1"
   },
   "build": {
-    "appId": "com.scanoss.quickscan",
-    "productName": "Quickscan",
+    "appId": "com.scanoss.quickscan.lite",
```
Comment (mscasso-scanoss) [inline package.json:35]: I would rather not change this one.

File: package.json [hunk 4]
```diff
@@ -53,12 +55,12 @@
       "vendor": "SCANOSS",
       "icon": "icons/",
       "category": "Utility",
-      "executableName": "scanoss-quickscan",
+      "executableName": "scanoss-quickscan-lite",
```
Comment (mscasso-scanoss) [inline package.json:58]: remove "scanoss-"

File: renderer.js [hunk 5]
```diff
@@ -165,6 +165,90 @@ function update_vuln_table(components) {
...
+            array[index] = array[index].trim(); //Remove any whitespace and add it to the set
+            if (!incompatible_licenses.has(array[index]))
+              incompatible_licenses.add(array[index]);
+          });
+        }
+
+      }
+
+      //Here is detected if there are a YES on copyleft field and change the text style to bold and red color
+      // let copyleft_element = $(row).children().eq(2);
```
Comment (mscasso-scanoss) [inline renderer.js:220]: Why is this commented? Please clean unnecessary comments.

File: scanner.js [hunk 6]
```diff
@@ -175,7 +182,8 @@ async function* walk(dir) {
     else if (
       d.isFile() &&
       !d.isSymbolicLink() &&
-      !winnowing.FILTERED_EXT.includes(path.extname(entry))
+      !winnowing.FILTERED_EXT.includes(path.extname(entry)) 
+      //stats.size>winnowing.MIN_FILE_SIZE
```
Comment (mscasso-scanoss) [inline scanner.js:186]: Again, be careful with comments

File: scanner.js [hunk 7]
```diff
@@ -185,17 +193,18 @@ async function recursive_scan(dir) {
   let wfp = '';
   let counter = 0;
   for await (const filepath of walk(dir)) {
-    counter++;
-    wfp += winnowing.wfp_for_file(
-      filepath,
-      filepath.replace(ctx.sourceDir, '')
-    );
+    //if(fs.lstatSync(filepath).size > winnowing.MIN_FILE_SIZE)
```
Comment (mscasso-scanoss) [inline scanner.js:196]: and here?? Why is this condition disabled?

**Review by mscasso-scanoss — CHANGES_REQUESTED** [review]

File: build/afterpack.js (hunk 1 above)
Comment (isasmendiagus) [reply build/afterpack.js:18]: This script is executed when compiling is called. The lines commented caused an inconsistent .desktop file after the compilation is done. 

This script was obtained from here and it is not neccesary for Quickscan
https://github.com/beekeeper-studio/beekeeper-studio/blob/master/build/afterPack.js

File: renderer.js (hunk 5 above)
Comment (isasmendiagus) [reply renderer.js:220]: It was a feature requested before and is not longer needed

File: scanner.js (hunk 7 above)
Comment (isasmendiagus) [reply scanner.js:196]: It was for a test. It should not be there

**Review by mscasso-scanoss — APPROVED** [review]

**Review by mscasso-scanoss — APPROVED** [review]

**Pull request merged**